nilearn/image/resampling.py

```diff
@@ -86,7 +86,7 @@ def from_matrix_vector(matrix, vector):
     return t
 
 
-def coord_transform(x, y, z, affine):
+def coord_transform(x, y, z, affine, squeeze=True):
```

Comment:
I wondered. From a developper POV, this is indeed the most wanted behavior. From a user POV, being able to transform only one point is nice.

Another option I thought of is this:
`coord_transform(1, 2, 3, np.eye(4))` returns a 3-tuple `(1, 2, 3)`
and
`coord_transform([1], [2], [3], np.eye(4))` returns a 3-tuple of list `([1], [2], [3])`

Maybe this is more natural?